src/limits.hpp

```diff
@@ -51,6 +51,35 @@ struct NumericLimits : public std::numeric_limits< T >
   static constexpr T denorm_min = std::numeric_limits< T >::denorm_min();
 };
 
+/**
+ * @struct NumericLimitsNC
```

Comment:
1. What does the NC stand for?
2. Do we want to have a preprocessor option to just alias the std::numeric_limits members? That way we can swap it out when we move to c++17?

Reply:
Non-constexpr.

I don't think it's a standards issue. The reason `NumericLimits` exists is so that you can have access to the value returned by the **host** function `std::numeric_limits::epsilon()`. The reason `NumericLimitsNC` exists is because occasionally if you capture a constexpr variable in a lambda and then do something like `EXPECT_LT( x, epsilon )` you'll get an undefined reference at link time to `epsilon`. I don't think this really has a use case outside of testing so I could move it there.

Reply:
that sounds like a nice option.